web/themes/custom/pineapple/sass/components/_field-label.scss

```diff
@@ -0,0 +1,17 @@
+.inline-field-label {
+  color: $primary;
+  padding: $unit;
+  margin-right: $unit;
+  display: inline;
+
+  .field__label,
+  .field__value {
+    display: inline;
+  }
+
+  .field__label {
+    padding-right: 0;
+    text-transform: uppercase;
+    font-weight: bolder;
+  }
+}
```

Comment:
Create a fields dir and add this file into it

Reply:
done